components/centos/centos-k8s-singlenode-setup/README.rst

```diff
@@ -12,7 +12,7 @@ QuickStart
 
 2. Create a directory for the Vagrant Box
 
-  ``$ mkdir directory && cd directory``
+  ``$ mkdir directory && cd $_``
```

Comment:
@ryanmaclean I think you want the Readme to have exact command so that you can copy and paste, right? If so we should give the directory a relevant name e.g. `$ mkdir adb_kubernetes && cd adb_kubernetes` . Keeping the directory name in the `cd` command makes it more readable for users who are not familiar with bash.